src/forms/DropPlaceHolder.js

```diff
@@ -65,59 +81,73 @@ const askTarget = {
       formDiagram.moveWidget(draggedItem.position, targetPosition);
     } else {
       formDiagram.appendWidget(draggedItem.field, targetPosition);
-      formDiagram.setState({ isHovering: false });
+      //formDiagram.setState({ isHovering: false });
     }
 
   }
 };
 
-
 @DropTarget('form_component', askTarget, (connect, monitor) => ({
   connectDropTarget: connect.dropTarget(),
   isOver: monitor.isOver()
 }))
 @connect(({ app, forms }) => ({ app, forms }))
 export default class DropPlaceHolder extends Component {
 
+  componentWillReceiveProps(nextProps) {
+    // This acts as an onLeave handler
+    if (this.props.isOver && !nextProps.isOver) {
+      this.props.formDiagram.enqueueReset();
+    }
+  }
+
   render() {
     return (
       this.props.connectDropTarget(
-        this.props.isOver ?
-          <div style={ styles.dropPlaceHolderActive }>
-          </div>
-        :
-          <div style={ styles.dropPlaceHolder }>
-            {
-              this.props.children ?
-                this.props.children
-              :
-                null /*<p style={ styles.emptyPlaceholderText }>Drag and drop fields here to add a question</p>*/
-            }
-          </div>
+        <div style={ styles.padder }>
+          {
+            this.props.isOver
+              ? <div style={ styles.dropPlaceHolderActive }></div>
+              : <div style={ styles.dropPlaceHolder }>
+                  {
+                    this.props.children ?
+                      this.props.children
+                    :
+                      <p style={ styles.emptyPlaceholderText }>Drag and drop fields here to add a question</p>
+                  }
+                </div>
+          }
+        </div>
       )
     );
   }
 }
 
 const styles = {
   dropPlaceHolder: {
-    minHeight: '60px',
-    background: 'rgba(128,128,128,.1)',
-    marginBottom: '10px',
-    borderRadius: '4px'
+    minHeight: '70px',
```

Comment:
px is the default measure in react/radium you can just use a number `minHeight: 70` here and in the following properties. It's not a merge blocker but just fyi